ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -824,10 +883,11 @@ private void initializeRefContainers() throws JSONException {
       refContainerPreviousEncryptionValues.add(previouslyEncrypted);
       refContainerCachingValues.add(random.nextBoolean());
       refContainerMediaScanDisabledValues.add(random.nextBoolean());
+      refContainerReplicationPolicyValues.add(UtilsTest.getRandomString(10));
```

Comment:
do you have a test for the null replicationPolicy case?

Reply:
Added